test/e2e/pd.go

```diff
@@ -42,6 +42,12 @@ var _ = Describe("PD", func() {
 	)
 
 	BeforeEach(func() {
+		// Special case for the local provider -- the tests will wound up being
+		// skipped, but we do not expect there to be multiple nodes.
+		if providerIs("local") {
```

Comment:
testContext.Provider shouldn't yet be filled in at this point in the code, so I would expect this to fail on all providers: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/test/e2e/util.go#L67

This check should only be done inside of an It() function.